src/EditorFeatures/TestUtilities/RefactoringHelpers/RefactoringHelpersTestBase.cs

```diff
@@ -20,20 +19,18 @@
 namespace Microsoft.CodeAnalysis.Editor.UnitTests.RefactoringHelpers
 {
     [UseExportProvider]
-    public abstract class RefactoringHelpersTestBase<TWorkspaceFixture> : TestBase, IClassFixture<TWorkspaceFixture>, IDisposable
+    public abstract class RefactoringHelpersTestBase<TWorkspaceFixture> : TestBase
         where TWorkspaceFixture : TestWorkspaceFixture, new()
     {
-        protected readonly TWorkspaceFixture fixture;
+        private readonly TestFixtureHelper<TWorkspaceFixture> _fixtureHelper = new();
 
-        protected RefactoringHelpersTestBase(TWorkspaceFixture workspaceFixture)
-            => this.fixture = workspaceFixture;
-
-        public override void Dispose()
+        protected RefactoringHelpersTestBase()
```

Comment:
nit: consider removing entirely.

Reply:
➡️ Covered by https://github.com/dotnet/roslyn/pull/47994#discussion_r494553088 (and discussion impacts several locations with the same characteristic)